src/recipes.js

```diff
@@ -46,11 +46,13 @@ const filterRecipesByIngredient = (recipes, searchedIngredient, ingredientData)
 };
 
 const searchRecipes = (allRecipes, allIngredients, userSearch) => {
-    const searchedItems = userSearch.split(' ')
-    const recipesByName = searchedItems.flatMap(searchWord => filterRecipesByName(allRecipes, searchWord))
-    const recipesByIngredient = searchedItems.flatMap(searchWord => filterRecipesByIngredient(allRecipes, searchWord, allIngredients))
-    const foundRecipes = [...recipesByName, ...recipesByIngredient]
-    return [...new Set(foundRecipes)]
+    const searchedItems = userSearch.split(' ').filter(word => word.length)
```

Comment:
oooooh nice!

Reply:
This way we don't have everything come up when someone searches word with a space after! :)

Reply:
YEP! LOVE IT